### DISCUSSION TITLE
Raspberry Pi 4 with Kodi 19 and working internal grabber for hyperion

### DISCUSSION THREAD

**Discussion by music-fan** [hyperion-project/hyperion.ng#1306] [Q&A] [answered]
Hi,
stupid question but important to me:
How can I get a reliable system running on a Pi 4 with Kodi 19 and Ambilight?

Since years, I'm using osmc OS on a raspberry Pi 3B+ with Kodi (now 18.9) and Hyperion. Now, I can't upgrade anymore, Hyperion is not Python3 and it looks like that even Hyperion.NG is - according forum messages - not working with the internal grabber on updated raspberry (Pi 3B+ as well as Pi 4 not working after upgrade to Kodi 19). Before I was using usb grabber, splitter and converter but this was a de-motivating experience; from other forum contributions I'm knowing this might work, even with limitations and audio problems, but anyway, I don't want go back; many devices broken, not compatible, ....

I'm asking for advice; how can I get my Pi 4 running Kodi 19 and using internal grabber to get the good old ambilight feeling?

Thanks

**Comment by Paulchen-Panther** [comment, marked as answer]
The number of programmers in the hyperion.ng team is very limited at the moment. Unfortunately, no new ones are found or some prefer to run their own forks. 🤣 
Joking aside. We are just about to take matters into our own hands. We ask for your patience. 

https://streamable.com/mv86vi

**Reply by music-fan** [reply]
Thank you. I'll be waiting patiently, old system is good working, I'm only missing USB3 and faster LAN speed. Toll, dass es Leute wie Euch beide gibt!

**Reply by webdandy** [reply]
Any news regarding Pi4 running on Matrix and ambilight?
